queries/python/highlights.scm

```diff
@@ -52,19 +52,6 @@
 (function_definition
   name: (identifier) @function)
 
-((function_definition
-   name: (identifier) @method
-   parameters: (parameters
-                 (identifier) @self) )
- (eq? @self "self"))
-
-((function_definition
-   name: (identifier) @constructor
-   parameters: (parameters
-                 (identifier) @self) )
- (eq? @self "self")
- (match? @constructor "(__new__|__init__)"))
-
```

Comment:
Why?

We explicitly added it because people wanted to have methods correctly highlighted. The purpose of this is not to highlight "@self" but to highlight methods/constructors.

Reply:
EDIT: you're probably right